src/pybind/mgr/rbd_support/schedule.py

```diff
@@ -164,7 +164,7 @@ def from_name(cls,
         if not namespace and image_name:
             name = "{}/{}".format(pool_name, image_name)
 
-        return LevelSpec(name, id, pool_id, namespace, image_id)
+        return LevelSpec(name, id, str(pool_id), namespace, image_id)
```

Comment:
What if it's a global level spec (i.e. if `pool_id` is `None`)?  This would pass `'None'` string and mess up the global level logic in `LevelSpec`.

Reply:
I'd suggest adding `pool_id = str(pool_id)` before line 128 instead.